core/designsystem/src/androidTest/java/com/google/samples/apps/nowinandroid/core/designsystem/ThemeTest.kt

```diff
@@ -241,6 +263,18 @@ class ThemeTest {
         )
     }
 
+    private fun defaultTintTheme(): TintTheme {
+        return TintTheme()
+    }
+
+    private fun dynamicTintTheme(colorScheme: ColorScheme): TintTheme {
```

Comment:
Nit: Maybe name this `dynamicTintThemeWithFallback` for consistency with the other methods?